source/lmp/plugin/deepmdplugin.cpp

```diff
@@ -41,6 +43,15 @@ extern "C" void lammpsplugin_init(void *lmp, void *handle, void *regfunc) {
   plugin.handle = handle;
   (*register_plugin)(&plugin, lmp);
 
+  plugin.version = LAMMPS_VERSION;
+  plugin.style = "pair";
+  plugin.name = "deepspin";
+  plugin.info = "deepspin pair style " STR_GIT_SUMM;
+  plugin.author = "Duo Zhang";
```

Comment:
This line may need to be reset after line 55 - note that we never re-initialize `plugin`, so the attributes in the previous plugin are kept.